plugin/src/main/java/at/helpch/chatchat/command/IgnoreListCommand.java

```diff
@@ -0,0 +1,40 @@
+package at.helpch.chatchat.command;
+
+import at.helpch.chatchat.ChatChatPlugin;
+import at.helpch.chatchat.api.user.ChatUser;
+import dev.triumphteam.cmd.bukkit.annotation.Permission;
+import dev.triumphteam.cmd.core.BaseCommand;
+import dev.triumphteam.cmd.core.annotation.Command;
+import dev.triumphteam.cmd.core.annotation.Default;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+
+import java.util.stream.Collectors;
+
+@Command("ignorelist")
+public class IgnoreListCommand extends BaseCommand {
+    private final ChatChatPlugin plugin;
+    private final static String IGNORELIST_PERMISSION = "chatchat.ignoreList";
+
+    public IgnoreListCommand(final ChatChatPlugin plugin) {
+        this.plugin = plugin;
+    }
+
+    @Permission(IGNORELIST_PERMISSION)
+    @Default
+    public void ignore(ChatUser sender) {
+        if (sender.ignoredUsers().isEmpty()) {
+            sender.sendMessage(plugin.configManager().messages().notIgnoringAnyone());
+            return;
+        }
+
+        String ignoredPlayers = sender.ignoredUsers()
+            .stream()
+            .map(Bukkit::getOfflinePlayer)
+            .map(OfflinePlayer::getName)
```

Comment:
I'm not quite sure this is a great idea. I've seen plugins like VoteParty run into performance issues doing similar stuff. I am not really educated on this tho so I'll wait for someone with more experience to confirm/deny this.

Yesterday when I was thinking about creating this PR myself, I thought about creating a UUID to Name dictionary like EssentialsX basically. I'm not sure how much better that is tho. Or if it even is better.

Reply:
Isn't spigot storing player uuid's (in the `usercache.json`)? That's why I'm using `OfflinePlayer` stuff

Reply:
`Bukkit.getOfflinePlayer(String)` possibly blocks the main thread as it would make network request to Mojang to check if the player with the name exists.

`Bukkit.getOfflinePlayer(UUID)` is safe to use as it just enquires the local cache and doesn't make any network request.